### DISCUSSION TITLE
CS8034 C# Unable to load Analyzer assembly {0}: {1}

### DISCUSSION THREAD

**Discussion by simonbrazer** [codewars/codewars.com#2456] [Q&A] [answered]
[
![129254496-3eedab68-2741-489c-8dad-5de9b57d97a3](https://user-images.githubusercontent.com/88495025/129451775-5efd0251-4122-4eba-9984-e440527539a2.png)
](url)
Hello.I'm trying to download these Packages from this video https://www.youtube.com/watch?v=C5cnZ-gZy2I timeframe in video 1:06:29 and only 1/4 installs succesfully,others fail.Tried seeing the problem in Github,but had no idea what people are talking about.Maybe someone could help?

**Comment by hunkyjimpjorps** [comment, marked as answer]
I'm not sure how you ended up here, but this is a discussion forum about operations on the site Codewars.com, not a place for general coding help or discussion.  You might have more luck asking in a C# or .NET community.

**Reply by simonbrazer** [reply]
And where are those communities in general? Github doesn't let me post anything for some reason?

**Reply by hunkyjimpjorps** [reply]
https://csharpforums.net/
https://www.reddit.com/r/csharp/
https://www.reddit.com/r/learncsharp/
https://discord.com/invite/ccyrDKv
https://stackoverflow.com/questions/tagged/c%23
and many other places you can find online.  I am not a C# user and I imply no warranty or obligation with these recommendations.
